provisioning/azuredeploy.json

```diff
@@ -1,446 +1,478 @@
-        "templateLink": {
-          "uri": "[concat(parameters('subtemplate location'), 'templates/OnlineTrainerEmptyTemplate.json')]",
-          "contentVersion": "1.0.0.0"
-        },
-        "parameters": {
-          "postfix": { "value": "[concat(variables('prefix'), reference('Microsoft.Resources/deployments/Unique_Postfix_Generator').outputs.FinalPassword.value)]" },
-          "location": { "value": "[variables('location')]" }
-        }
-      }
-    },
-    {
-      "apiVersion": "2015-01-01",
-      "name": "Online_Trainer_Deployment",
-      "type": "Microsoft.Resources/deployments",
-      "dependsOn": [
-        "Microsoft.Resources/deployments/Admin_Token_Generator",
-        "Microsoft.Resources/deployments/Storage",
-        "Microsoft.Resources/deployments/Application_Insights",
-        "Microsoft.Resources/deployments/Event_Hub_Joined",
-        "Microsoft.Resources/deployments/Event_Hub_Evaluation",
-        "Microsoft.Resources/deployments/SAS_Uri_Generator"
-      ],
-      "properties": {
-        "mode": "incremental",
-        "templateLink": {
-          "uri": "[concat(parameters('subtemplate location'), 'templates/OnlineTrainerTemplate.json')]",
-          "contentVersion": "1.0.0.0"
-        },
-        "parameters": {
-          "onlineTrainerCspkgToken": { "value": "[reference('Microsoft.Resources/deployments/SAS_Uri_Generator').outputs.SASTokenOnlineTrainerCspkg.value]" },
-          "configLink": { "value": "[concat(reference('Microsoft.Resources/deployments/Management_Center').outputs.url.value, '/Deployment/GenerateTrainerConfig?key=', reference('Microsoft.Resources/deployments/Storage').outputs.userStorageAccountKey.value)]" },
-          "name": { "value": "[reference('Microsoft.Resources/deployments/Online_Trainer').outputs.name.value]" },
-          "location": { "value": "[variables('location')]" }
-        }
-      }
-    }
-  ],
-  "outputs": {
-    "Management Center URL": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/Management_Center').outputs.url.value]"
-    },
-    "Management Center Password": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/Management_Password_Generator').outputs.FinalPassword.value]"
-    },
-    "Client Library URL": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/SAS_Uri_Generator').outputs.SASTokenUriClientSettings.value]"
-    },
-    "Web Service Token": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/Web_Service_Token_Generator').outputs.FinalPassword.value]"
-    },
-    "Online Trainer Token": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/Admin_Token_Generator').outputs.FinalPassword.value]"
-    },
-    "Online Trainer URL": {
-      "type": "string",
-      "value": "[reference('Microsoft.Resources/deployments/Online_Trainer').outputs.url.value]"
-    }
-  }
-}
+﻿{

```

Comment:
looks like there are some inconsistent whitespaces here, can you fix? It's hard to tell the difference otherwise.